admyral/config/config.py

```diff
@@ -58,6 +62,33 @@ def get_local_postgres_volume() -> str:
     return os.path.join(get_local_storage_path(), "postgres")
 
 
+def get_user_config_file() -> str:
+    return os.path.join(get_global_project_directory(), GLOBAL_CONFIG_FILE)
+
+
+def get_user_id() -> str:
+    config_file = get_user_config_file()
+    if os.path.exists(config_file):
+        with open(config_file, "r") as f:
+            file_content = yaml.safe_load(f)
+            return file_content["user_id"]
+    else:
+        user_id = str(uuid.uuid4())
+        with open(config_file, "w") as f:
+            yaml.dump({"user_id": user_id, "posthog_disabled": False}, f)
+        return user_id
+
+
+def read_posthog_disabled() -> bool:
+    config_file = get_user_config_file()
+    if os.path.exists(config_file):
+        with open(config_file, "r") as f:
+            file_content = yaml.safe_load(f)
+            return file_content["posthog_disabled"]
+
+    return True
```

Comment:
I don't think we need two functions. I think `load_local_config` should be responsible for reading and initializing the config file and populating `GlobalConfig`.